### ISSUE TITLE
Road Blocks 2 "requires Flash 9 or higher" 

### ISSUE THREAD

**Issue by Poopooracoocoo** [ruffle-rs/ruffle#2319]
https://www.newgrounds.com/portal/view/398903?emulate=flash

![image](https://user-images.githubusercontent.com/51363864/103905834-38157d80-5153-11eb-8415-60d5eb06e5aa.png)

**Comment by Toad06** [comment]
Thanks for the report.

This issue also happens in the official Flash Player. Version detection has actually been done in a poor way:

```
playerVersion = System.capabilities.version;
majorVersion = playerVersion.substr(i + 1, 1); // returns "3" instead of "32" (the current Flash version), as this code only gets the first character.
if(majorVersion < 8) // "3" < 8
{
   stop(); // we're trapped.
}
```

Closing in favor of #819 as this is the enhancement feature we'll need here.

**Comment by n0samu** [comment]
The enhancement has been implemented in #8961.
